src/components/Header/Header.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import './Header.scss';
 import tomatilloIcon from '../../greenTomato.png'
+import { useNavigate } from 'react-router-dom'
 
-const Header = () => {
+const Header = ({ setModalIsOpen }) => {
+  const navigate = useNavigate()
+  
+  function handleBackArrowClick() {
+    setModalIsOpen(false);
```

Comment:
Ah! I think this what you were saying is what fixed the issue we were facing yesterday

Reply:
Just you wait, your mind will be blown with what the simple solution was .... it was in app.js I believe (without looking at it)